Mathlib/Geometry/Manifold/ContMDiffMFDeriv.lean

```diff
@@ -286,9 +286,9 @@ theorem ContMDiffOn.contMDiffOn_tangentMapWithin
     (hf : ContMDiffOn I I' n f s) (hmn : m + 1 ≤ n)
     (hs : UniqueMDiffOn I s) :
     ContMDiffOn I.tangent I'.tangent m (tangentMapWithin I I' f s)
-      (π E (TangentSpace I) ⁻¹' s) := by
```

Comment:
Can you omit these changes for now, please? (I'm not sure if the notation should be changed instead.)

Reply:
Done